lerna.json

```diff
@@ -0,0 +1,14 @@
+{
+  "packages": ["packages/*"],
+  "useWorkspaces": true,
+  "npmClient": "yarn",
+  "command": {
+    "publish": {
+      "conventionalCommits": true
+    },
+    "version": {
```

Comment:
use `allow-branch` option to safeguard the release flow
- https://github.com/lerna/lerna/tree/main/commands/version#--allow-branch-glob